test/contract.test.ts

```diff
@@ -0,0 +1,66 @@
+import { expect } from "chai";
+import { starknet } from "hardhat";
+import { TIMEOUT } from "./constants";
+import { StarknetContractFactory } from "hardhat/types/runtime";
+import { getPredeployedOZAccount } from "./util";
+import { OpenZeppelinAccount } from "@shardlabs/starknet-hardhat-plugin/dist/src/account";
+import * as fs from "fs";
+import { StarknetContract } from "@shardlabs/starknet-hardhat-plugin/dist/src/types";
+
+describe("StarknetContractFactory", function () {
+    this.timeout(TIMEOUT);
+    let contractFactory: StarknetContractFactory;
+    let account: OpenZeppelinAccount;
+    let contract: StarknetContract;
+    let initial_balance = 25n;
+
+    /**
+ * invoke(functionName: string, args?: StringMap, options?: InvokeOptions): Promise<InvokeResponse>;
```

Comment:
What about this piece of commented code?

Reply:
Woops, this was like.., my checklist...